assets/src/scss/new-identity/style.scss

```diff
@@ -1,14 +1,26 @@
 @import "../base/tokens";
+@import "../base/variables";
+@import "../base/mixins";
 
 @mixin shared-link-styles {
+  --link--height: 26px;
+  --transition--duration: 0.3s;
+
   border-bottom: 2px solid var(--gp-green-500);
-  transition: 0.3s cubic-bezier(0.5, 0, 0, 1);
+  transition: var(--transition--duration) cubic-bezier(0.5, 0, 0, 1);
+
+  &:hover {
+    box-shadow: inset 0 calc(var(--link--height) * -1) rgba($gp-green-500, .15);
+  }
 
   &:hover,
-  &:focus {
-    box-shadow: inset 0 calc(-22px) var(--gp-green-100);
+  &:active {
     text-decoration: none !important;
   }
+
+  &:focus {
+    --link--focus--border-radius: 0;
```

Comment:
Since it's also used [here](https://github.com/greenpeace/planet4-master-theme/blob/8027e00a203eca3f55da7f47274ae9abf1aed936/assets/src/scss/layout/_country-selector.scss#L20). What do you think?

Reply:
I think that in both places it should be used in the `:focus-visible` selector instead of `:focus` 🤔 And then that should solve the problem

Reply:
I assume it's not only to replace to `:focus-visible` but also to re-think this part too.
```css
  button:focus {
    border-radius: var(--link--focus--border-radius, 4px);
    ...
  }

  button:focus:not(:focus-visible) {
    ...
  }
```

To be honest, I don't like to touch the current behavior of other blocks, especially when they are not part of the ticket's scope. In most of the cases I'd prefer to keep the new identity styles more cleaner with the newest changes, including overrides, and whenever we'll merge it then analyze the part of replacing the newest code with the oldest one, but it's up to you! Does it make sense?

Reply:
Yes, you're right, we can think about this later! Maybe for now for the new identity it's fine to use `!important` once in a while.